.vscode/settings.json

```diff
@@ -1,71 +0,0 @@
-{
-  "files.exclude": {
```

Comment:
Did you intend to remove this file?

Reply:
No - I think I accidentally made these changes